packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -106,16 +106,24 @@ const _useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedP
     if (isScreenShareActive) {
       return [];
     }
+    // if we have no video participants we need to cap the max number of audio participants in the grid
+    // we will use the max streams provided to the function to find the max participants that can go in the grid
+    // if there are less participants than max streams then we will use all participants including joining in the grid
     /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
     return visibleGridParticipants.current.length > 0
```

Comment:
I think this could be a little cleaner. Maybe
```suggestion
return visibleGridParticipants.current.length > 0
      ? visibleGridParticipants.current
      : maxRemoteVideoStreams && visibleOverflowGalleryParticipants.current.length > maxRemoteVideoStreams
      ? visibleOverflowGalleryParticipants.current.slice(0, maxRemoteVideoStreams)
      : visibleOverflowGalleryParticipants.current.concat(callingParticipants);
```